gwlfe/WriteOutputFiles.py

```diff
@@ -7,9 +7,717 @@
+            z.n2 = z.AvLuSedYield[l]
+            z.n6 = z.AvLuTotNitr[l]
+            z.n13 = z.AvLuTotPhos[l]
+            z.n6dn = z.AvLuDisNitr[l]
+            z.n13dp = z.AvLuDisPhos[l]
+            z.n24 = round(z.Area[l])
+        elif (z.Landuse[l].lower() == "cropland"):
+            z.n1 = z.AvLuSedYield[l]
+            z.n5 = z.AvLuTotNitr[l]
+            z.n12 = z.AvLuTotPhos[l]
+            z.n5dn = z.AvLuDisNitr[l]
+            z.n12dp = z.AvLuDisPhos[l]
+            z.n23 = round(z.Area[l])
+        elif (z.Landuse[l].lower() == "turfgrass"):
+            z.n2t = z.AvLuSedYield[l]
+            z.n6t = z.AvLuTotNitr[l]
+            z.n13t = z.AvLuTotPhos[l]
+            z.n24t = round(z.Area[l])
+        elif (z.Landuse[l].lower() == "unpaved_road"):
+            z.n2d = z.AvLuSedYield[l]
+            z.n6d = z.AvLuTotNitr[l]
+            z.n13d = z.AvLuTotPhos[l]
+            z.n6ddn = z.AvLuDisNitr[l]
+            z.n13ddp = z.AvLuDisPhos[l]
+        else:
+            AvOtherLuSed = AvOtherLuSed + z.AvLuSedYield[l]
+            AvOtherLuNitr = AvOtherLuNitr + z.AvLuTotNitr[l]
+            AvOtherLuPhos = AvOtherLuPhos + z.AvLuTotPhos[l]
+
+    LoadReductions.AdjustScnLoads(z)
+
+    z.n2c = 0
+    z.n6c = 0
+    z.n13c = 0
+    z.n24b = 0
+    z.n2b = 0
+    z.n6b = 0
+    z.n13b = 0
+    z.n23b = 0
+    z.n6cdn = 0
+    z.n13cdp = 0
+    z.n6bdn = 0
+    z.n13bdp = 0
+
+    for l in range(z.NRur + 1, z.NLU):
+        if "ld" in z.Landuse[l].lower():
+            z.n2c = z.n2c + z.AvLuSedYield[l]
+            z.n6c = z.n6c + z.AvLuTotNitr[l]
+            z.n13c = z.n13c + z.AvLuTotPhos[l]
+            z.n6cdn = z.n6cdn + z.AvLuDisNitr[l]
+            z.n13cdp = z.n13cdp + z.AvLuDisPhos[l]
+            z.n24b = z.n24b + round(z.Area[l])
+        elif "hd" in z.Landuse[l].lower() or "md" in z.Landuse[l].lower():
+            z.n2b = z.n2b + z.AvLuSedYield[l]
+            z.n6b = z.n6b + z.AvLuTotNitr[l]
+            z.n13b = z.n13b + z.AvLuTotPhos[l]
+            z.n6bdn = z.n6bdn + z.AvLuDisNitr[l]
+            z.n13bdp = z.n13bdp + z.AvLuDisPhos[l]
+            z.n23b = z.n23b + round(z.Area[l])
+
+    # Initial Upland loads
+    InitialUplandN = z.n5 + z.n6 + z.n6b + z.n6c + z.n6d + AvOtherLuNitr
+    if InitialUplandN == 0:
+        InitialUplandN = 0.00000000001  # Fix for Divide-by-Zero error
+    InitialUplandP = z.n12 + z.n13 + z.n13b + z.n13c + z.n13d + AvOtherLuPhos
+    if InitialUplandP == 0:
+        InitialUplandP = 0.00000000001  # Fix for Divide-by-Zero error
+    InitialUplandSed = z.n1 + z.n2 + z.n2b + z.n2c + z.n2d + AvOtherLuSed
+    if InitialUplandSed == 0:
+        InitialUplandSed = 0.00000000001  # Fix for Divide-by-Zero error
+
+    # FOR POINT SOURCE
+    YrPointNitr = 0
+    YrPointPhos = 0
+    for i in range(0, 12):
+        YrPointNitr = YrPointNitr + z.PointNitr[i]
+        YrPointPhos = YrPointPhos + z.PointPhos[i]
+
+    # GET THE AVERAGE SEPTIC SYSTEM INFORMATION
+    if z.SepticFlag == 1:
```

Comment:
`SepticFlag` and several other fields are actually "enums" as well https://github.com/WikiWatershed/gwlf-e/blob/develop/gwlfe/parser.py#L197.

This comparison should be `if z.SepticFlag is enums.YesOrNo.YES`. 

I realize this is a really bad name for an enum...